src/app/lib/streamer.js

```diff
@@ -1,501 +1,491 @@
-(function (App) {
+(function (App, WebTorrent) {
```

Comment:
here is why: https://github.com/butterproject/butter-desktop/blob/master/.jshintrc#L80
i didn't put webtorrent on .jshintrc